composer.json

```
@@ -14,24 +14,24 @@
         "issues": "https://github.com/acquia/http-hmac-php/issues"
     },
     "require": {
-        "php": ">=7.3.0",
+        "php": "^7.3 || ^7.4 || ^8.0 || ^8.1",
```

The original constraint already supported all these versions. To the extent that you want to avoid supporting a theoretical PHP 9, then you only need to do:


```suggestion
        "php": "^7.3 || ^8.1",
```

The carrot operator constrains the allowed versions up to the next major version: `^8.1` means `>=8.1.0` and `<9.0.0`.

---

Yeah, initially I set it up like that but then composer couldn't build on 7.4.* or 8.0.*. So will set it as this instead:
`        "php": ">=7.3.0 <8.2",`
This way it should build from 7.3.x to 8.1.x and we can keep all those tests working too.